### PR TITLE
M3-1479 Rename Oauth clients tab to 'My Apps'

### PR DESCRIPTION
**Pull request by WilkinsKa1**
I kept scope for this ticket to exclusively handle display and some QA verification instances, as I didn't want to change/break every instance that relies on the oauth naming pattern. Let me know if I missed any instances that should be included for this!

### CODE REVIEW

**Comment by johnwcallahan** [comment]
Should "Create an OAuth Client" be updated as well, or is that out of the scope of this ticket? There are a few other instances that are just labels, e.g. before you reset a token the language is "Are you sure you want to permanently reset the secret of this OAuth client?". 
screen shot 2018-10-15 at 7 58 35 am

**Comment by martinmckenna** [comment]
^^^ Was literally just about to comment this. I agree we should be consistent across the board.

The term _OAuth Client_ appears in all of the following areas

1. Creating an Oauth Client
2. Editing an OAuth Client
3. Deleting an OAuth Client
4. Resetting an OAuth Client secret

**Comment by WilkinsKa1** [comment]
Thanks @martinmckenna and @johnwcallahan for the feedback! I went back and edited the rest of the instances under 'Profile'. I left out the QA and 'API Tokens' references.

File: src/features/Profile/OAuthClients/OAuthClientActionMenu.tsx [hunk 1]
```diff
@@ -160,7 +160,7 @@ class OAuthClientActionMenu extends React.Component<CombinedProps> {
           onClose={this.closeConfirmReset}
         >
           <Typography>
-            Are you sure you want to permanently reset the secret of this OAuth client?
+            Are you sure you want to permanently reset the secret of this app?
```
Comment (Jskobos) [inline src/features/Profile/OAuthClients/OAuthClientActionMenu.tsx]: Unrelated to this PR but 'of' should be 'for', or "reset this app's secret" maybe

**Pull request merged**